tests/test_frame.py

```diff
@@ -160,6 +161,17 @@ def test_types_getitem() -> None:
     df[i]
 
 
+def test_types_getitem_with_hashable() -> None:
+    # Testing getitem support for hashable types that are not scalar
+    # Due to the bug in https://github.com/pandas-dev/pandas-stubs/issues/592
+    class MyEnum(Enum):
+        FIRST = "tayyar"
+        SECOND = "haydar"
+
+    df = pd.DataFrame(data = [[12.2, 10], [8.8, 15]], columns=[MyEnum.FIRST, MyEnum.SECOND])
+    df[MyEnum.FIRST]
```

Comment:
Can you use `check(assert_type(df[MyEnum.FIRST], pd.Series), pd.Series)`

Reply:
Thanks I just added it.